src/aws/lambda-update-state/src/updateVideosContentType.js

```diff
@@ -0,0 +1,26 @@
+const AWS = require('aws-sdk');
+const url = require('url');
+
+const s3 = new AWS.S3();
+
+module.exports = async (videos) => {
+  for (const video of videos) {
+    const origin = video.outputFilePaths[0];
+    const parsedUrl = url.parse(origin);
+
+    const params = {
+      Bucket: parsedUrl.host,
+      CopySource: `/${parsedUrl.host}${parsedUrl.path}`,
+      Key: parsedUrl.path.substring(1),
+      ContentType: 'binary/octet-stream',
+      MetadataDirective: 'REPLACE'
+    }
+
+    await s3.copyObject(params, (err, data) => {
+      if(err) {
+        console.log(`An error occured trying to update metadata of ${origin}`);
```

Comment:
why not "trying to update content-type to binary/octet-stream" ?

Reply:
This does not look like a failure message. I should add a console to know that we are trying to update the content-type ?

Reply:
Done